cmd/operator-sdk/bundle/cmd.go

```diff
@@ -44,10 +44,22 @@ Operator Lifecycle Manager:
 https://github.com/operator-framework/operator-lifecycle-manager
 `,
 	}
+	return cmd
+}
 
+func NewCmdLegacy() *cobra.Command {
+	cmd := newCmd()
 	cmd.AddCommand(
 		newCreateCmd(),
```

Comment:
the newCreateCmd is just used for the legacy one. Could we add a deprecated note on that as well?

Reply:
So, IMO the deprecated comment would be more important to be used in the funcs which are just used by `NewCmdLegacy()` and has no utility for the new layout at all. If it is just used for the legacy one then,  in POV would make sense we deprecated it. 

PS.:  I understand that we agreed in the -2 ago eng. meeting use the Go deprecation annotations for all that is just used for the legacy one unless it make visible to the users since you are right we did not officially deprecate them. The pros here are let us know very easy what should or not receiving a new logic, what is or not just used for the legacy and etc in order to keep the code maintained and to help us clean up all when the legacy one is no longer be supported. Note that it makes very visible because shows a strikethrough on the func in the IDEs as well.

Reply:
So, IMO the deprecated comment would be more important to be used in the funcs which are just used by `NewCmdLegacy()` and has no utility for the new layout at all. If it is just used for the legacy one then,  in POV would make sense we deprecated it.